docs/CMIP7/guidance_for_modellers.md

```diff
@@ -213,6 +226,51 @@ For testing purposes sample CV JSON files are available via the [CMOR tables](ht
 
 Examples of the input JSON file for CMOR are available via [a jupyter notebook](https://github.com/WCRP-CMIP/cmip7-cmor-tables/blob/main/cmor_demo.ipynb).
 
+### 6b cmip7repack
```

Comment:
Could you combine the section 6c with this one -- I think of these as two tools from the same package and it would be good to minimise the number of sections here.